src/test/resources/test.properties

```diff
@@ -12,3 +21,14 @@ voyage.security.test.clients.standard-client.client-secret=secret
 
 # Voyage Security: Brute Force Attack (BFA)
 voyage.security.brute-force-attack.sleep-after-failure.enabled=false
+
+# Voyage Security: Email default configuration
+spring.mail.host=localhost
+spring.mail.port=3025
+spring.mail.overrideAddress=testmsg@LighthouseSoftware.com
+spring.autoconfigure.exclude= org.springframework.cloud.aws.autoconfigure.mail.MailSenderAutoConfiguration
+
+# Voyage Security: Email Template
```

Comment:
These props are getting duplicated from application.properties - by default these should come from application, unless the user wants to override them for tests